impeller/entity/shaders/glyph_atlas_sdf.vert

```diff
@@ -0,0 +1,39 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+uniform FrameInfo {
+  mat4 mvp;
+} frame_info;
+
+in vec2 unit_vertex;
```

Comment:
This seems to be shared with the non-SDF glyph atlas implementation. You can put the common stuff in its own header and #include that here and in the other spot too.

Reply:
TBH, I am surprised so much is shared between the two strategies.

Reply:
This is very similar. We might be able to simplify it if we move SDF to using a fixed glyph size, but it's probably not worth it. The differences are really in the fragment shader.

Reply:
Moved this to transforms.glsl